packages/manager/modules/hub/src/components/user-panel/template.html

```diff
@@ -0,0 +1,45 @@
+<div class="manager-hub-user-panel">
+    <div class="mb-3">
+        <ovh-manager-hub-user-infos
+            data-me="$ctrl.me"
+            data-support-level="$ctrl.supportLevel"
+        >
+        </ovh-manager-hub-user-infos>
+    </div>
+    <div class="mb-4">
+        <ovh-manager-hub-payment-mean> </ovh-manager-hub-payment-mean>
+    </div>
+    <div class="mb-4">
+        <ovh-manager-hub-shortcuts data-notifications="$ctrl.notifications">
+        </ovh-manager-hub-shortcuts>
+    </div>
+    <div class="manager-hub-user-panel_links">
+        <strong data-translate="hub_links_title"></strong>
+        <hr />
```

Comment:
This is not the semantic use of `hr` I think: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr